java-components/sidecar/src/main/java/com/redhat/hacbs/sidecar/resources/deploy/containerregistry/Gav.java

```diff
@@ -0,0 +1,91 @@
+package com.redhat.hacbs.sidecar.resources.deploy.containerregistry;
+
+import java.util.Objects;
+
+public class Gav {
+    private String groupId;
```

Comment:
I know it does not really matter here but these should really be immutable. In general mutability + equals/hashCode can result in lots of problems, as the hash code would change if you modify the object.